### PR TITLE
fix(starfish) Span summary panel QOL improvements

### PR DESCRIPTION
**Pull request by gggritso**
Mostly just tooltips and labels and so on.

- Add span metric tooltips
- Add app impact metric
- Improve labels and calculations

### CODE REVIEW

File: static/app/views/starfish/views/spans/spanSummaryPanel/index.tsx [hunk 1]
```diff
@@ -37,28 +41,42 @@ export function SpanSummaryPanel({span, onClose}: Props) {
...
           <TimeSince date={spanMetrics?.last_seen} />
           {lastSeenSpanEvent?.release && (
             <ReleasePreview release={lastSeenSpanEvent?.release} />
           )}
         </Block>
 
-        <Block title={t('Total Spans')}>{spanMetrics?.count}</Block>
+        <Block
+          title={t('Total Spans')}
+          description={t('The total number of times this span was seen in all time')}
```
Comment (0Calories) [inline static/app/views/starfish/views/spans/spanSummaryPanel/index.tsx:68]: In the current retention window, right?

File: static/app/views/starfish/views/spans/spanSummaryPanel/index.tsx [hunk 2]
```diff
@@ -37,28 +41,42 @@ export function SpanSummaryPanel({span, onClose}: Props) {
...
-        <Block title={t('Total Time')}>
-          <Duration
-            seconds={spanMetrics?.total_time / 1000}
-            fixedDigits={2}
-            abbreviation
-          />
+        <Block
+          title={t('App Impact')}
+          description={t(
+            'The total exclusive time taken up by this span vs. entire application'
```
Comment (0Calories) [inline static/app/views/starfish/views/spans/spanSummaryPanel/index.tsx:76]: Should we say "exclusive" time here? I have a feeling this is just grounds for more confusion

**Review by 0Calories — APPROVED** [review]
Noice 👌

**Comment by gggritso** [comment]
@0Calories I'll follow up with Dora on the official text for the tooltips 🙏🏻

**Pull request merged**